bin/config.py

```diff
@@ -5,20 +5,20 @@
 import os
 import re
 from pathlib import Path
-from typing import Final, Literal, Optional
+from typing import Final, Literal, Mapping, Optional, Sequence
 
 # return values
-RTV_AC: Final = 42
-RTV_WA: Final = 43
+RTV_AC: Final[int] = 42
+RTV_WA: Final[int] = 43
 
-SUBMISSION_DIRS: Final = [
+SUBMISSION_DIRS: Final[Sequence[str]] = [
```

Comment:
I assume that `Sequence` is used here instead of `list`, because a `list` is mutable and a `Sequence` is not? Nice :smile:

Note however that `typing.Sequence` is a deprecated alias to `collections.abc.Sequence`: https://docs.python.org/3/library/typing.html#typing.Sequence

(same goes for `collections.abc.Mapping`)